### DISCUSSION TITLE
Failed to retrieve tenants. Perform a permission check ()

### DISCUSSION THREAD

**Discussion by Exigo-CIPP** [KelvinTegelaar/CIPP#1495] [Q&A]
Any help is appreciated.'

![CIPP](https://user-images.githubusercontent.com/120699528/233246307-cb0175ba-0120-459d-9f67-d7e142f93d6f.jpg)

**Comment by rvdwegen** [comment]
Re-run the SAM wizard.

**Reply by Exigo-CIPP** [reply]
Hi Roel,

Which option requires to be selected? I have tried with "I would like to refresh my tokens or replace the user I've used for my previous tokens." and waiting for 24 hours to update.

Any other suggestion ?

**Reply by rvdwegen** [reply]
Get on the discord and post in #issues. that's a bit more practical.
